DreamJournal/DreamJournal/HomeView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  HomeView.swift
+//  DreamJournal
+//
+//  Created by CUBS Customer on 11/4/22.
+//
+
+import SwiftUI
+import CoreData
+
+struct HomeView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+    
+    @FetchRequest(entity: Entry.entity(), sortDescriptors: [NSSortDescriptor(key: "date", ascending: false)]) private var allEntries: FetchedResults<Entry>
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
+                LinearGradient(gradient: Gradient(colors: [CustomColor.Navy, CustomColor.SkyPurple]), startPoint: .top, endPoint: .bottom)
+                    .ignoresSafeArea()
+                Image("MoonLogo")
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width:300, height: 300)
+                VStack {
+                    Divider()
+                    Spacer()
+                    List {
+                        ForEach(allEntries) {
+                            entry in HStack {
```

Comment:
Minor style note — it's a little funny to have the `entry in HStack` on the same line.

Typically, you'd have `{ entry in` on the same line as the ForEach. I noticed that elsewhere too.

Reply:
I can see where that would look a bit strange-- I'll be sure to fix that!